styles/components/_notice.scss

```diff
@@ -0,0 +1,24 @@
+.notice {
+  background-color: $focus;
+  width: 90%;
```

Comment:
```suggestion
  width: 100%;
```
Might look nicer full width, IMO.

Reply:
Moved to bottom of the page and kept at the original widths.